plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -73,27 +72,38 @@
 import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
 
 @TestInstance(PER_CLASS)
-public class BaseTestHiveOnDataLake
+abstract class BaseTestHiveOnDataLake
         extends AbstractTestQueryFramework
 {
-    private static final String HIVE_TEST_SCHEMA = "hive_datalake";
-    private static final DataSize HIVE_S3_STREAMING_PART_SIZE = DataSize.of(5, MEGABYTE);
+    protected static final String HIVE_TEST_SCHEMA = "hive_datalake";
+    protected static final DataSize HIVE_S3_STREAMING_PART_SIZE = DataSize.of(5, MEGABYTE);
 
-    private String bucketName;
-    private Hive3MinioDataLake hiveMinioDataLake;
-    private HiveMetastore metastoreClient;
+    protected String bucketName;
+    protected HiveMinioDataLake hiveMinioDataLake;
+    protected HiveMetastore metastoreClient;
+
+    @BeforeAll
+    public void setUp()
```

Comment:
Why did you move this method? The original place looks good to me. This setup should happen after `createQueryRunner()` method.

Reply:
It was by mistake.